### ISSUE TITLE
v1.20 not working together with autofill forms 1.1.0.1

### ISSUE THREAD

**Issue by kruess** [sarahavilov/secure-login#10]
After the latest secure-login update, using the icon or the hotkey does not fill in the credentials, anymore. It only asks for the credentials to use (if there is more than one), but that's it.

Disabling autofill forms will solve it, but I'd like to use both together.

BfN, Konrad
(FF 45.1.0 ESR)

**Comment by sarahavilov** [comment]
Please try 1.2.1 and let me know if it fixes the problem or not; https://addons.mozilla.org/en-US/firefox/addon/secure-login/versions

**Comment by kruess** [comment]
Yep, that looks much better: Visually, it looks like after pushing the
hotkeys it clears the username and just fills in the password, but the
login works properly. Additional, now it also presses 'enter' again
after filling in the credentials - something your add-on did not do
anymore since few months.

Thanks for the quick response!

Regards,
Konrad
